README.md

```diff
@@ -10,24 +10,22 @@ This theme is pretty basic and covers all of the essentials. All you have to do
 
 ---
 
-- [Hello Friend](#hello-friend)
-    - [DEMO - https://hugo-hello-friend.now.sh/](#demo---httpshugo-hello-friendnowsh)
-  - [Features](#features)
-      - [Built-in shortcodes](#built-in-shortcodes)
-      - [Code highlighting](#code-highlighting)
-      - [Improved RSS Feed](#improved-rss-feed)
-  - [How to start](#how-to-start)
-  - [How to configure](#how-to-configure)
-  - [How to add a cover image to your posts](#how-to-add-a-cover-image-to-your-posts)
-  - [How to display the Last Modified Date in your posts](#how-to-display-the-last-modified-date-in-your-posts)
-  - [How to hide "Read more" button](#how-to-hide-%22read-more%22-button)
-  - [Add-ons](#add-ons)
-  - [How to run your site](#how-to-run-your-site)
-  - [How to edit the theme](#how-to-edit-the-theme)
-  - [How to contribute](#how-to-contribute)
-  - [`Hello Friend` theme user?](#hello-friend-theme-user)
-  - [Sponsoring](#sponsoring)
-  - [License](#license)
+- [Features](#features)
+    - [Built-in shortcodes](#built-in-shortcodes)
+    - [Code highlighting](#code-highlighting)
+    - [Improved RSS Feed](#improved-rss-feed)
+- [How to start](#how-to-start)
+- [How to configure](#how-to-configure)
+- [How to add a cover image to your posts](#how-to-add-a-cover-image-to-your-posts)
+- [How to display the Last Modified Date in your posts](#how-to-display-the-last-modified-date-in-your-posts)
+- [How to hide "Read more" button](#how-to-hide-read-more-button)
+- [Add-ons](#add-ons)
+- [How to run your site](#how-to-run-your-site)
+- [How to edit the theme](#how-to-edit-the-theme)
+- [How to contribute](#how-to-contribute)
+- [`Hello Friend` theme user?](#hello-friend-theme-user)
+- [Sponsoring](#sponsoring)
+- [License](#license)
```

Comment:
It looks like we have different settings for TOC generation. I guess I'm ignoring H1s. Do you want me to add those back in, or is this fine?